src/gazebo_ros_image_sonar.cpp

```diff
@@ -0,0 +1,855 @@
+/*
```

Comment:
Are these copyright notices still appropriate?

Reply:
I will erase them in the next commit